### PR TITLE
Feature/new models

### PR DESCRIPTION
**Pull request by Programmer-yar**
This PR contains migrations which should only be applied to clean/new database.
Run the following commands to clear existing db:
```
python manage.py migrate socialaccount zero
python manage.py migrate sites zero
python manage.py migrate sessions zero
python manage.py migrate auth zero
python manage.py migrate contenttypes zero
python manage.py migrate admin zero
python manage.py migrate account zero
python manage.py migrate polls zero
```
Run the migrations again using:
```
python manage.py migrate
```
Create admin user:
```
python manage.py createsuperuser
```

### CODE REVIEW

File: polls/models.py [hunk 1]
```diff
@@ -1,3 +1,54 @@
...
+class Batch(models.Model):
+    created_at = models.DateTimeField()
+    is_gold = models.BooleanField(default=False)
+    notes = models.TextField()
+
+    class Meta:
+        verbose_name_plural = "batches"
+
+
+class CurrentBatch(models.Model):
```
Comment (turian) [inline polls/models.py:19]: This is wrong. We can't have multiple `CurrentBatch` rows. But this model allows that. Is there another way to hold these global variables in the database? Or can you add appropriate constraints so there is always only one row??

File: polls/models.py [hunk 2]
```diff
@@ -1,3 +1,54 @@
+from tabnanny import verbose
 from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.contrib.auth import get_user_model
 
-# Create your models here.
+class User(AbstractUser):
```
Comment (turian) [inline polls/models.py:6]: How does this work with the oauth stuff?

Comment (Programmer-yar) [reply polls/models.py:6]: This basically extends the builtin User table in Django. It has the same role as base User model.

File: polls/models.py [hunk 3]
```diff
@@ -1,3 +1,54 @@
+from tabnanny import verbose
```
Comment (turian) [inline polls/models.py:1]: Uhhh what is this and why did you add it? You didn't see the warnings from LGTM that this import is unused?

Comment (Programmer-yar) [reply polls/models.py:1]: Sorry, this came from VS code auto import, I have disabled it and will also remove this import in next commit

Comment (turian) [reply polls/models.py]: @Programmer-yar Please review your PR diffs before asking for review. Thank you.

Comment (Programmer-yar) [reply polls/models.py]: Alright

**Review by turian — APPROVED** [review]
